backend/docker-compose-postgres.yml

```diff
@@ -30,3 +30,4 @@ services:
       - "db:database"
     ports:
       - "${PGADMIN_PORT:-8080}:80"
+      
```

Comment:
Have you delete the spaces? I see that you change this online, but you commit something locally without pulling this change, so this is not modified int the end(this is why you use force-push with `-f` param).

You should pull your branch first and avoid force push(only use `git push -f` when necessary)